bin/celery_test_worker.py

```diff
@@ -0,0 +1,33 @@
+# script for checking readiness + liveness of celery worker
```

Comment:
Maybe it could be good to add that it can be used in Kubernetes and add an example of how to run the command (like in the article) in the comment?